src/interaction/encode.ts

```diff
@@ -25,12 +28,13 @@ export default function(type: Event): void {
         case Event.TouchEnd:
         case Event.TouchMove:
         case Event.TouchCancel:
-            for (let i = 0; i < pointer.data[type].length; i++) {
-                let entry = pointer.data[type][i];
-                tokens = [entry.time, type];
-                tokens.push(observe(entry.target));
-                tokens.push(entry.x);
-                tokens.push(entry.y);
+            for (let i = 0; i < pointer.data.length; i++) {
+                if (task.blocking(timer)) { await task.idle(timer); }
```

Comment:
>blocking [](start = 25, length = 8)

What are you doing in this code (the handler for touch cancel)? Why is this different from all other events?